cpp/ReadMe.md

```diff
@@ -0,0 +1,24 @@
+# Quick Base C++ Craft demo
```

Comment:
@sergioharo added a readme. also added a stretch goal of writing DeleteRecord to see the usage of the collection.